provision/acc_provision/acc_provision.py

```diff
@@ -383,11 +410,26 @@ def config_adjust(args, config, prov_apic, no_random):
     if args.version_token:
         token = args.version_token
 
-    if extern_static:
+    if config["flavor"] != "calico-3.23.2" and extern_static:
```

Comment:
If we end up supporting more Calico operator version, and hence creating more flavors, you will have to keep updating this every time. Instead, in the flavor defintion, you can perhaps add a configuration which indicates that the flavor deals with Calico CNI, and then above check with be only for that configuration, as opposed to specific strings. When a new Calico flavor gets added, we just need ensure that the configuration is present, and no code changes will be required here and all the other places where such a check is being performed.